packages/react-image/src/components/Image/Image.tsx

```diff
@@ -1,10 +1,22 @@
-import { compose, createClassResolver } from '@fluentui/react-compose';
-
-import { ImageBase } from './ImageBase';
+import * as React from 'react';
 import { ImageProps } from './Image.types';
+import { useFocusRects } from '@uifabric/utilities';
+import { makeClasses } from '@fluentui/react-compose/lib/next/index';
```

Comment:
```suggestion
import { makeClasses } from '@fluentui/react-compose/lib/next';
```

?

Reply:
This is how the import was used in all other components, I would suggest if we decide to change the imports to do it on all components at the same time..

Reply:
Background: if we use file path imports without resolving to a physical file, it breaks consumers who use AMD bundlers like r.js which don't automatically infer index.js when refering to a path.